src/watchlists/WatchListItem.jsx

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+import styles from './styles';
+import tradeIcon from '../../assets/icons/watchList/trade.png';
+import detailIcon from '../../assets/icons/watchList/detail.png';
+
+
+export default function watchListItem({
+  stockName,
+  curPrice,
+  fluctuation,
+  dealAmount,
+}) {
+  return (
+    <View style={styles.watchListItem}>
+      <View style={styles.watchListItemColumn}>
+        <Text style={[styles.watchListItemColumnText, { fontWeight: 'bold', color: '#566ed3' }]}>
+          {stockName}
+        </Text>
+      </View>
+      <View style={styles.watchListItemColumn}>
+        <Text style={styles.watchListItemColumnText}>
+          {curPrice}
+        </Text>
+      </View>
+      <View style={styles.watchListItemColumn}>
+        <Text style={[styles.watchListItemColumnText, { color: fluctuation > 0 ? '#21af78' : '#d82f2f' }]}>
+          {`${fluctuation > 0 ? '+' : ''}${fluctuation}%`}
+        </Text>
```

Comment:
please consider the following:
```jsx
fluctuation > 0 ? ( 
  <Text style={[styles.watchListItemColumnText, { color: '#21af78' }]}>
    {`+${fluctuation}%`}
  </Text>
) : (
  <Text style={[styles.watchListItemColumnText, { color: '#d82f2f' }]}>
    {`${fluctuation}%`}
  </Text>
)
```
Also you can create a sub-component for this if you want to make it more cleaner